<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1072</width>
    <height>595</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>566</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CLCheckers 2.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="playerOneView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>211</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QGraphicsView" name="rivalView">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>211</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>430</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Новая игра</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>430</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Загрузить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>430</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Сохранить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="surrenderButton">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>460</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Сдаться</string>
    </property>
   </widget>
   <widget class="QLabel" name="youLab">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>191</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;
text-decoration: underline;</string>
    </property>
    <property name="text">
     <string>Вы</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="rivalIndName">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>191</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;
text-decoration: underline;</string>
    </property>
    <property name="text">
     <string>Противник</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="offerDrawButton">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>460</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Предложить ничью</string>
    </property>
   </widget>
   <widget class="QPushButton" name="controlsButton">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>490</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Управление</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ruleButton">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>490</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Правила</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="boardView">
    <property name="geometry">
     <rect>
      <x>157</x>
      <y>27</y>
      <width>490</width>
      <height>490</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QLabel" name="playerName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Тут никого нет</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="rivalName">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>320</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Тут никого нет</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="gameLog">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>60</y>
      <width>256</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="gameLogLabel">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>30</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;
text-decoration: underline;</string>
    </property>
    <property name="text">
     <string>Ход партии:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1072</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
